810-well-known-custom-extensions.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Well known custom extensions
+weight: 810
+---
+
+# Well known custom actions
```

Comment:
Also actions => extensions 😄

Reply:
It looks we don't currently hyphenate in https://github.com/cnabio/cnab-spec/blob/b5ed2f30bdc51fb9953fcd74d6b13d83c4165849/804-well-known-custom-actions.md#well-known-custom-actions

Reply:
I strongly prefer the hyphenated form. It bothered me as soon as I saw it, but I stuck with `well known` only because of precedent. Since I've got your "vote" too I'm going to use `well-known` here and we can change the other to use the hyphen in another PR.